fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -209,7 +209,7 @@ def self.keychain_path(name)
       #
 
       # Remove the ".keychain" at the end of the name
-      name.sub!(/\.keychain$/, "")
+      name = name.sub(/\.keychain$/, "")
```

Comment:
ahh i see the prev. code changed the reference variable.
however, how about changing the local name of `name`  instead of turning the input variable into a local variable?

Reply:
Suggestion?

Reply:
Yeah, just using a new variable instead of assigning it to the same one should solve the issue 👍

Reply:
`local_name`, `keychain_name`  i am not 100% sure 😵

Reply:
I renamed the method param to `keychain_name` as this is actually what is goven to the method I think, then by removing the `.keychain` it is only a name we ued to expand to `keychain_paths` again.